package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "project-guava",
-  "version": "1.0.0",
+  "version": "0.1.0",
   "description": "The project repository for an open system to generate and manage certificates.",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "test": "mocha test/*.test.js",
     "format": "prettier --write './**/*.js'",
     "lint": "eslint --fix './**/*.js'",
```

Comment:
Add a script for lint check without fix one. To see what all errors are there

Reply:
yup yup, will add one